korgw/src/jvmMain/kotlin/com/soywiz/korgw/platform/INativeGL.kt

```diff
@@ -2,164 +2,187 @@ package com.soywiz.korgw.platform
 
 import com.soywiz.kmem.FBuffer
 import com.soywiz.korim.bitmap.NativeImage
+import com.sun.jna.NativeLong
 import java.nio.ByteBuffer
 import java.nio.FloatBuffer
 import java.nio.IntBuffer
 
-typealias IntSize = Long
+typealias GLboolean = Boolean
```

Comment:
Due to JNA behavior, this can't be a bool. We had a bug related to this with some drivers on linux. Explanation here:
https://github.com/korlibs/korge/issues/268#issuecomment-729056184

Reply:
I believe it should be Byte, and explicitly use 1 or 0